Makefile

```
@@ -39,6 +39,15 @@ OUT   = obj
 USE_WAYLAND = OFF
 
 
+# Is this macOS?
+IS_MACOS = 0
+ifneq ($(OS),Windows_NT)
+ifeq ($(shell uname -s),Darwin)
```

```suggestion
 ifeq ($(shell uname -s),Darwin)
  MACOS = ON
 endif
```
Note: using single spaces for functional indents, would want nested ifs to be indented too.
Also note this suggestion does not span all relevant lines.